src/tests/firmwarecontrol/Firmware_Upgrade_Upgrade_002.test.js

```diff
@@ -0,0 +1,34 @@
+import { firmwareUpgrade } from '../../commonMethods/firmwarecontrol'
+import { pluginActivate, pluginDeactivate } from '../../commonMethods/controller'
+import constants from '../../commonMethods/constants'
+
+export default {
+  title: 'Firmware Upgrade - 002',
+  description: 'Check the upgrade Functionality when upgrade is already in progress',
+  setup() {
+    return this.$sequence([
+      () => pluginDeactivate.call(this, constants.firmwareUpgradePlugin),
+      () => pluginActivate.call(this, constants.firmwareUpgradePlugin),
+    ])
+  },
+  steps: [
+    {
+      description:
+        'Invoke Upgrade firmware when already firmware upgrade in progres and validate the result',
+      sleep: 5,
+      test() {
+        //TODO - Update the test to enter the 'name', 'location', 'type', 'progressinterval', 'hmac' through User prompt
+        firmwareUpgrade.call(this, 'name', 'location', 'type', 'progressinterval', 'hmac')
+        return firmwareUpgrade.call(this, 'name', 'location', 'type', 'progressinterval', 'hmac')
```

Comment:
Suggest to use promise.all to run 2 promises in parallel, for reference:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all

And same here, guess we wait until we can fill in something meaninful to avoid false negative results?